src/app/api/community/malysis/reports/[analysisId]/route.ts

```diff
@@ -0,0 +1,33 @@
+import { apiErrorHandler } from "@/lib/api/error";
+import { createMalwareAnalysisServiceClient } from "@/lib/rpc/client";
+import { NextApiRequest } from "next";
+import { NextResponse } from "next/server";
+
+async function handleGET(
+  _req: NextApiRequest,
+  { params }: { params: Promise<{ analysisId: string }> },
+) {
+  const analysisId = (await params).analysisId;
+  if (!analysisId) {
+    throw new Error("Analysis ID is required");
+  }
+
+  const communityTenantId = process.env.COMMUNITY_API_TENANT_ID as string;
+  const communityApiKey = process.env.COMMUNITY_API_KEY as string;
+
+  if (!communityTenantId || !communityApiKey) {
+    throw new Error("Community API credentials are required");
+  }
+
+  const client = createMalwareAnalysisServiceClient(
+    communityTenantId,
+    communityApiKey,
+  );
+  const report = await client.getAnalysisReport({
+    analysisId: analysisId,
+  });
+
+  return NextResponse.json(report.toJson());
```

Comment:
@kunaldevxxx Here we serialize our `report` (object) into JSON using its in-built method suitable for serializing Protocol Buffers object. We will then load the same in frontend. This avoids the need to manually define type.